resource_obmcs_objectstorage_object.go

```diff
@@ -94,6 +157,18 @@ func (s *ObjectResourceCrud) Update() (e error) {
 	content := s.D.Get("content").(string)
 	opts := &baremetal.PutObjectOptions{}
 
+	if contentEncoding, ok := s.D.GetOk("content_encoding"); ok {
```

Comment:
Doesn't need to be for this PR, but why don't we just mark everything ForceNew and get rid of Update? I don't believe that objects have a real update - you must send the full set of data every time, and that overwrites what is there.